src/store.cc

```diff
@@ -369,7 +369,7 @@ StoreEntry::kickProducer()
 void
 StoreEntry::destroyMemObject()
 {
-    debugs(20, 3, HERE << "destroyMemObject " << mem_obj);
+    debugs(20, 3, "destroyMemObject " << mem_obj);
```

Comment:
Agreed, but this PR automatically removes HERE. Nothing else should be required. Removal of function names is discussed in another comment so I will not repeat those arguments here.